javascript/javascript3/week1/giphy.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="giphy.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>GIPHY SEARCHER</h1>
+    <input type="text" placeholder="giphy name" id="giphy-name" /><br />
+    <button>search</button><br />
```

Comment:
button type should we submit, as when it will get click the event automatically get triggered